src/org/pentaho/reporting/platform/plugin/connection/PentahoJndiConnectionReadHandler.java

```diff
@@ -44,29 +44,26 @@ public PentahoJndiConnectionReadHandler() {
 
   /**
    * Returns the handler for a child element.
-   * 
-   * @param tagName
-   *          the tag name.
-   * @param atts
-   *          the attributes.
+   *
+   * @param tagName the tag name.
+   * @param atts    the attributes.
    * @return the handler or null, if the tagname is invalid.
-   * @throws SAXException
-   *           if there is a parsing error.
+   * @throws SAXException if there is a parsing error.
    */
   protected XmlReadHandler getHandlerForChild( final String uri, final String tagName, final Attributes atts )
-    throws SAXException {
+      throws SAXException {
     if ( isSameNamespace( uri ) == false ) {
       return null;
     }
-    if ( "path".equals( tagName ) ) { //$NON-NLS-1$
+    if ( "path" .equals( tagName ) ) { //$NON-NLS-1$
```

Comment:
I applied the automatic rules for check style, but I'll reverse those specific changes.